### PR TITLE
Migrate MatomoOptOut to Chakra

### PR DESCRIPTION
**Pull request by ImMiguelP**
## Description
Migrates `MatomoOptOut` component from `emotion` to `chakra-ui`

## Related Issue
#6374

### CODE REVIEW

File: src/components/MatomoOptOut.tsx [hunk 1]
```diff
@@ -53,31 +31,43 @@ const MatomoOptOut: React.FC<IProps> = () => {
...
+    <Flex
+      border="1px solid"
+      borderColor="border"
+      bgColor="background"
+      borderRadius={1}
+      p={6}
+      direction="column"
+      mb={4}
+      mt={8}
+      alignItems="flex-start"
```
Comment (pettinarip) [inline src/components/MatomoOptOut.tsx:43]:
```suggestion
      align="flex-start"
```

File: src/components/MatomoOptOut.tsx [hunk 2]
```diff
@@ -53,31 +31,43 @@ const MatomoOptOut: React.FC<IProps> = () => {
     localStorage.setItem(MATOMO_LS_KEY, String(!checked))
   }
   return (
-    <Container>
-      <p>
+    <Flex
+      border="1px solid"
+      borderColor="border"
+      bgColor="background"
+      borderRadius={1}
```
Comment (pettinarip) [inline src/components/MatomoOptOut.tsx:38]:
```suggestion
      borderRadius="base"
```

File: src/components/MatomoOptOut.tsx [hunk 3]
```diff
@@ -53,31 +31,43 @@ const MatomoOptOut: React.FC<IProps> = () => {
...
+      p={6}
+      direction="column"
+      mb={4}
+      mt={8}
+      alignItems="flex-start"
+      justifyContent="space-between"
```
Comment (pettinarip) [inline src/components/MatomoOptOut.tsx:44]:
```suggestion
      justify="space-between"
```

File: src/components/MatomoOptOut.tsx [hunk 4]
```diff
@@ -53,31 +31,43 @@ const MatomoOptOut: React.FC<IProps> = () => {
...
         prevent us from learning from your actions and creating a better website
         experience for you and other users.
-      </p>
+      </Text>
       {loading ? (
         "Loading preferences..."
       ) : (
-        <form>
-          <input
+        <FormControl m={0}>
```
Comment (pettinarip) [inline src/components/MatomoOptOut.tsx:55]: not sure if this is working as expected. In https://ethereumorgwebsitedev01-immiguelpethereumorgwebsi61534.gtsb.io/en/cookie-policy/ I don't see the input
image

**Review by pettinarip — CHANGES_REQUESTED** [review]

File: src/components/MatomoOptOut.tsx [hunk 5]
```diff
@@ -53,31 +31,41 @@ const MatomoOptOut: React.FC<IProps> = () => {
...
-            checked={!isOptedOut}
-            onChange={handleCheckbox}
-          />
-          <label htmlFor="matomo">
-            {isOptedOut
-              ? "You are opted out. Check this box to opt-in."
-              : "You are not opted out. Uncheck this box to opt-out."}
-          </label>
-        </form>
+        <Checkbox
```
Comment (pettinarip) [inline src/components/MatomoOptOut.tsx:55]: This is great, gj on using the `Checkbox` here.

File: src/components/MatomoOptOut.tsx (hunk 2 above)
Comment (pettinarip) [reply src/components/MatomoOptOut.tsx:38]: Why didn't we include this? we want to use the `base` radius here (`4px`), right?

Comment (ImMiguelP) [reply src/components/MatomoOptOut.tsx:38]: Apologies  I thought I had pushed the file that with base in the borderRadius will fix the styling right now

**Review by pettinarip — APPROVED** [review]
Well done @ImMiguelP 💪🏼 ty

**Pull request merged**